sdk/spring/tests.yml

```diff
@@ -84,6 +80,8 @@ stages:
           AAD_SINGLE_TENANT_CLIENT_SECRET: $(java-spring-aad-single-tenant-secret)
           AAD_SINGLE_TENANT_CLIENT_ID_WITH_ROLE: $(java-spring-aad-single-tenant-id-with-role)
           AAD_SINGLE_TENANT_CLIENT_SECRET_WITH_ROLE: $(java-spring-aad-single-tenant-secret-with-role)
+          AAD_LOGIN_BASE_URL: 'https://login.microsoftonline.com/'
+          AAD_GRAPH_BASE_URL: 'https://graph.microsoft.com/'
```

Comment:
Can we delete the 2 variables, and use `AZURE_CLOUD_TYPE` instead? the supported values are `Global` and `China`.

Reply:
Yes, we can add one single EnvVar of `AZURE_CLOUD_TYPE` to replace these two vars, but we cannot use the `Clouds` parameter in the below comment, because it is defined in another yaml, so I will write it as AZURE_CLOUD_TYPE: 'Public' or AZURE_CLOUD_TYPE: 'China' here. Is that OK?

Reply:
OK.

Maybe we should use `Global/China`, not `Public/China`.
Refs: https://docs.microsoft.com/en-us/azure/china/resources-developer-guide#check-endpoints-in-azure